packages/ui-utils/src/hash.ts

```diff
@@ -64,6 +64,45 @@ function executeHash(input: string): string {
   return toBase64(String(hash))
 }
 
+// resource: https://stackoverflow.com/a/52171480
+function cyrb53(str: string, seed = 0) {
+  let h1 = 0xdeadbeef ^ seed,
+    h2 = 0x41c6ce57 ^ seed
+  for (let i = 0, ch; i < str.length; i++) {
+    ch = str.charCodeAt(i)
+    h1 = Math.imul(h1 ^ ch, 2654435761)
+    h2 = Math.imul(h2 ^ ch, 1597334677)
+  }
+  h1 =
+    Math.imul(h1 ^ (h1 >>> 16), 2246822507) ^
+    Math.imul(h2 ^ (h2 >>> 13), 3266489909)
+  h2 =
+    Math.imul(h2 ^ (h2 >>> 16), 2246822507) ^
+    Math.imul(h1 ^ (h1 >>> 13), 3266489909)
+
+  return (
+    (h2 >>> 0).toString(16).padStart(8, '0') +
+    (h1 >>> 0).toString(16).padStart(8, '0')
+  )
+}
+function hashInstance(instance: string, map: Map<string, number>) {
+  if (!map.has(instance)) {
+    map.set(instance, 0)
+  } else {
+    map.set(instance, map.get(instance)! + 1)
+  }
+
+  const instanceCount = map.get(instance)!
+  const valueToHash = `${instance}_${instanceCount}`
+  const hashed = cyrb53(valueToHash)
```

Comment:
why do we need to hash this? Inst just `instanceName_count` enough to make it unique and deterministic?

Reply:
You are correct, I've removed it, thanks!